lib/Importer/ONNXModelLoader.cpp

```diff
@@ -837,6 +852,50 @@ llvm::Error ONNXModelLoader::loadPad(const ONNX_NAMESPACE::NodeProto &op,
   return llvm::Error::success();
 }
 
+llvm::Error ONNXModelLoader::loadCast(const ONNX_NAMESPACE::NodeProto &op,
+                                      const ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+
+  // Input type
+  NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input,
+                             getNodeValueOrCreateConstantByName(op.input(0)));
+  ElemKind inputKind = input.getType()->getElementType();
+
+  // Target type.
+  ElemKind targetKind;
+  if (dict.count("to")) {
```

Comment:
nit: you could replace this if with `RETURN_ERR_IF_NOT(dict.count("to"), "Cast: missing 'to' attribute");`

Reply:
Done.